src/Directory.Build.props

```diff
@@ -24,13 +23,14 @@
         <DefaultLanguage>en-US</DefaultLanguage>
         <IncludeSymbols>false</IncludeSymbols>
         <IsTestProject>$(MSBuildProjectName.Contains('Test'))</IsTestProject>
-        <LangVersion>12</LangVersion>
+        <LangVersion>latest</LangVersion>
```

Comment:
I assume you're aware but from the docs:

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version

> [!WARNING] 
> 
> Setting the LangVersion element to latest is discouraged. The latest setting means the installed compiler uses its latest version. That can change from machine to machine, making builds unreliable. In addition, it enables language features that may require runtime or library features not included in the current SDK.

Reply:
Very aware, thanks